extractor/src/main/resources/all-projects.yml

```diff
@@ -42,11 +42,31 @@ projects:
       - '**/test/langtools/tools/javac/unicode/UnicodeAtEOL.java'
       - '**/test/langtools/tools/javac/unicode/UnicodeCommentDelimiter.java'
       - '**/test/langtools/tools/javac/unicode/FirstChar2.java'
+      - '**/test/langtools/tools/javac/processing/model/util/elements/TestGetDocComment_Line.java'
+      - '**/test/langtools/tools/javac/diags/examples/FeatureFlexibleConstructors.java'
+      - '**/test/langtools/tools/javac/diags/examples/PreviewFeatureUse.java'
+      - '**/test/langtools/tools/javac/patterns/UninitializedThisException.java'
+      - '**/test/langtools/tools/javac/SuperInit/LambdaOuterCapture.java'
+      - '**/test/langtools/tools/javac/SuperInit/EarlyLocalTest1.java'
+      - '**/test/langtools/tools/javac/SuperInit/EarlyLambdaReturn.java'
+      - '**/test/langtools/tools/javac/SuperInit/EarlyLocalTest2.java'
+      - '**/test/langtools/tools/javac/SuperInit/EarlyLocalCtorRef.java'
+      - '**/test/langtools/tools/javac/SuperInit/EarlyLocalTest4.java'
+      - '**/test/langtools/tools/javac/SuperInit/EarlyLocalTest5.java'
+      - '**/test/langtools/tools/javac/SuperInit/EarlyLocalTest3.java'
+      - '**/test/langtools/tools/javac/SuperInit/SuperInitGood.java'
+      - '**/test/langtools/tools/javac/diags/examples/ImplicitClass.java'
+      - '**/test/langtools/tools/javac/ImplicitClass/NestedEnum.java'
+      - '**/test/langtools/tools/javac/flags/NoFalseSealedError.java'
       - '**/test/langtools/tools/javac/diags/examples/UnnamedClass.java'
       - '**/test/jdk/java/lang/Class/UnnamedClass/Unnamed.java'
       - '**/test/langtools/tools/javac/unnamedclass/SourceLevelErrorPosition.java'
       - '**/test/langtools/tools/javac/processing/model/element/Anonymous.java'
       - '**/test/langtools/tools/javac/unnamedclass/NestedEnum.java'
+      - '**/test/langtools/tools/javac/diags/examples/ImportModule.java'
+      - '**/test/langtools/tools/javac/diags/examples/ImportModuleDoesNotReadUnnamed.java'
+      - '**/test/langtools/tools/javac/diags/examples/ImportModuleDoesNotRead/test/Test.java'
+      - '**/test/langtools/tools/javac/diags/examples/ImportModuleNotFound.java'
```

Comment:
FYI, I have added those. I assume we first want to run the diff report CI job and ensure it passes. 

Then we remove those, ensure diff report CI fails, merge this PR, run diff report CI on https://github.com/checkstyle/checkstyle/pull/18079, ensure it passes and we finally merge https://github.com/checkstyle/checkstyle/pull/18079?

Maybe I am overcomplicating it..